scala/scala.bzl

```diff
@@ -551,6 +551,9 @@ def _collect_jars_from_common_ctx(ctx, extra_deps = [], extra_runtime_deps = [])
 
     return struct(compile_jars = cjars, transitive_runtime_jars = transitive_rjars, jars2labels=jars2labels, transitive_compile_jars = transitive_compile_jars)
 
+def _format_full_jars_for_intellij_plugin(full_jars):
+    return [struct (class_jar = file, ijar = None) for file in files if not _is_source_jar(file)]
```

Comment:
weird spacing here, right. Why not `struct(class_jar = file, ijar = None)`?

Reply:
I think `files` here should be `full_jars`; this doesn't run right now.

Reply:
yep. Looks like CI caught that.

Reply:
Oscar,
Didn't understand what you said about spacing. Can you take a look at the current version and comment again if it's still an issue?